### DISCUSSION TITLE
My email was removed by someone and i lost accoutn access

### DISCUSSION THREAD

**Discussion by ata-hastam** [community/community#127246] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello
I received an email that your email was removed from GitHub, moments later I received an email that a new email was added.
Then I couldn't log in to my GitHub account.
My account has two-step login and I have the passcodes.
Please help to restore my account because it contains very important information and repositories.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @ataaminie ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by ata-hastam** [reply]
I am very upset because my account information is very critical.
Please tell me if there is a way to get my account back

**Comment by SUDEEP-M-SHETTY** [comment]
Contact GitHub support as soon as possible. Explain the situation in detail, including the unauthorized email changes and your inability to log in. Provide any evidence you have, such as the emails you received.

GitHub support will likely ask you to verify your identity, so have your 2FA passcodes ready. They may also request additional information to confirm your ownership of the account.

Once your identity is verified, GitHub support should be able to restore your account and revert the unauthorized email changes.
After regaining access, review your account settings and activity to ensure no other suspicious changes have been made.

**Reply by ata-hastam** [reply]
Yes, I sent a ticket, but I haven't received a message yet.
Is there no way to restore?
I have the recovery codes

**Comment by SUDEEP-M-SHETTY** [comment]
GitHub provides an automated recovery process if you have lost access to your account. This involves going to the "Reset your password" option and entering the email associated with your account. You should then receive an email with steps to reset your password.

**Reply by ata-hastam** [reply]
Yes, I did this, the problem is that he deleted my email from my GitHub account and replaced it with another email, and now GitHub doesn't recognize my user password.

received emails:
[GitHub] Your password has changed
[GitHub] An email address was added to your account.
[GitHub] An email address was removed from your account.

last email about my email address

**Reply by SUDEEP-M-SHETTY** [reply]
I'm sorry to say this but there isn't anything much that you can do now bro. Try recovering that email id first if possible.

**Reply by SUDEEP-M-SHETTY** [reply]
@ataaminie if can still see that GitHub account try to fork those repo, you can still access those repo if they are not set to private.

**Reply by ata-hastam** [reply]
My access has been lost
this is another account
all my repositories were private
i have filed a ticket, but the support team has not responded yet

**Comment by toeisk** [comment]
I got hit like this too, which can't do anything, sent to github support, didn't get any feedback back, and recently my account couldn't find it any more :(

ticket: #2816689

**Comment by ata-hastam** [comment]
Please visit https://github.com/orgs/community/discussions/127293
